osf/models/mixins.py

```diff
@@ -2129,11 +2132,15 @@ def suspend_spam_user(self, user):
         # Make public nodes private from this contributor
         for node in user.all_nodes:
             if self._id != node._id and len(node.contributors) == 1 and node.is_public and not node.is_quickfiles:
+                if set_spam:
+                    node.confirm_spam(save=True)
                 node.set_privacy('private', log=False, save=True)
 
         # Make preprints private from this contributor
         for preprint in user.preprints.all():
             if self._id != preprint._id and len(preprint.contributors) == 1 and preprint.is_public:
+                if set_spam:
+                    preprint.confirm_spam(save=True)
                 preprint.set_privacy('private', log=False, save=True)
```

Comment:
what's our value in marking it as private but not spam? seems harder to find and un-spam things that get incorrectly flagged this way. If we spam the content by default, we can fix everything with a confirm ham call on the flagged user.

Reply:
The value is in not training Akismet on things that should otherwise not be spam -- if a spammer perfectly mimics a legitimate object in terms of `spam_fields`, but gets around that somehow, we'd be training Akismet to find a bunch of false positives

Reply:
Are we keeping Akismet alongside OOPSpam? And are we training OOPSpam the same way?

I'm fine with switching to `confirm_spam` as a follow-on fix and adding a param to skip any training step. As is, though, we're adding an (unreliable) DevOps step to undoing any falsely-flagged users by just marking things private, since Eric et al have no way of identifying and recovering all of the disappeared resources (this was A Thing last Friday).

Reply:
Right now Akismet is still running alongside OOPSpam. OOPSpam does not have training like Akismet does. Removing akismet and switching to confirm_spam perhaps should be a follow up ticket.